mentoring/public/js/mentoring_assessment_view.js

```diff
@@ -279,12 +279,16 @@ function MentoringAssessmentView(runtime, element, mentoring) {
     }
 
     function handleSubmitResults(response){
+        submitDOM.attr('disabled','disabled');
```

Comment:
Doesn't `handleResults` already do this? And why are we disabling the button here, if we're already doing it when the user clicks it?